syntax = "proto3";

enum Status {
    FAILED = 0;
    SUCCESS = 1;
  }

message RegisterServerRequest {
    string server_name = 1;
    string ip = 2;
    int32 port = 3;
  }

message RegisterServerResponse {
    Status status = 1;
  }

message GetServerListRequest {
    string client_uuid = 1;
}

message GetServerListResponse {
    map<string, string> servers = 1;
  }